go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -142,7 +145,7 @@ func registerTabletEnvFlags(fs *pflag.FlagSet) {
 	flagutil.DualFormatBoolVar(fs, &currentConfig.EnableTxThrottler, "enable_tx_throttler", defaultConfig.EnableTxThrottler, "If true replication-lag-based throttling on transactions will be enabled.")
 	flagutil.DualFormatStringVar(fs, &currentConfig.TxThrottlerConfig, "tx_throttler_config", defaultConfig.TxThrottlerConfig, "The configuration of the transaction throttler as a text formatted throttlerdata.Configuration protocol buffer message.")
 	flagutil.DualFormatStringListVar(fs, &currentConfig.TxThrottlerHealthCheckCells, "tx_throttler_healthcheck_cells", defaultConfig.TxThrottlerHealthCheckCells, "A comma-separated list of cells. Only tabletservers running in these cells will be monitored for replication lag by the transaction throttler.")
-	fs.Var((*topoproto.TabletTypeListFlag)(&currentConfig.TxThrottlerTabletTypes), "tx-throttler-tablet-types", "A comma-separated list of tablet types. Only tablets of this type are monitored for replication lag by the transaction throttler. Supported types are replica and/or rdonly.")
+	fs.StringSliceVar(&txThrottlerTabletTypes, "tx-throttler-tablet-types", []string{"replica"}, "A comma-separated list of tablet types. Only tablets of this type are monitored for replication lag by the transaction throttler. Supported types are replica and/or rdonly.")
```

Comment:
> the default value should just be defaultConfig.TxThrottlerTabletTypes

@ajm188 that works, but it would mean the struct-field `TxThrottlerTabletTypes` needs to become `[]string`

The drawback is we'd probably need to parse strings to tablet-types twice, once in `.verifyTxThrottlerConfig()` to validate the config early, then again when we compare `topodata.TabletType` in `.StatsUpdate()` (in `*txThrottlerState`). Do you see a smoother way than that?